### DISCUSSION TITLE
Canonical Examples

### DISCUSSION THREAD

**Discussion by HyperCrowd** [PsySecGroup/foundation#6] [Ideas]
Psychosecurity can benefit from having easy-to-understand examples and quotes to help cognitively onboard people who are interested in it.  Reply with such examples so we can suss them out.

And yes, we recognize quote citations are an appeal to authority/nostalgia framing (known psychosecurity exploits) :P

**Comment by HyperCrowd** [comment]
[“When one with honeyed words but evil mind Persuades the mob, great woes befall the state.” ― Euripides/Orestes](https://www.goodreads.com/quotes/414438-when-one-with-honeyed-words-but-evil-mind-persuades-the)

**Comment by HyperCrowd** [comment]
[And thus I clothe my naked villany](https://www.enotes.com/shakespeare-quotes/thus-clothe-my-naked-villany)
[With odd old ends stol'n out of holy writ,](https://www.enotes.com/shakespeare-quotes/thus-clothe-my-naked-villany)
[And seem a saint, when most I play the devil.](https://www.enotes.com/shakespeare-quotes/thus-clothe-my-naked-villany)
~ William Shakespeare, _Richard III_

**Comment by HyperCrowd** [comment]
[“I'm not upset that you lied to me, I'm upset that from now on I can't believe you.”  ― Friedrich Nietzsche](https://www.goodreads.com/quotes/11864-i-m-not-upset-that-you-lied-to-me-i-m-upset)

**Comment by HyperCrowd** [comment]
[“The best way to find out if you can trust somebody is to trust them.”  ― Ernest Hemingway](https://www.goodreads.com/quotes/217785-the-best-way-to-find-out-if-you-can-trust)

**Comment by HyperCrowd** [comment]
[Abraham Lincoln is falsely and ironically credited with saying "You can fool some of the people all of the time, and all of the people some of the time, but you can not fool all of the people all of the time."](https://quoteinvestigator.com/2013/12/11/cannot-fool/)

**Comment by AirFireWater** [comment]
“Therefore it is unnecessary for a prince to have all the good qualities I have enumerated, but it is very necessary to appear to have them. And I shall dare to say this also, that to have them and always to observe them is injurious, and that to appear to have them is useful; to appear merciful, faithful, humane, religious, upright, and to be so, but with a mind so framed that should you require not to be so, you may be able and know how to change to the opposite.”
― Niccolò Machiavelli, The Prince

**Comment by AirFireWater** [comment]
“Ignorance of naturall causes disposeth a man to Credulity, so as to believe many times impossibilities: for such know nothing to the contrary, but that they may be true; being unable to detect the Impossibility. And Credulity, because men love to be hearkened unto in company, disposeth them to lying: so that Ignorance it selfe without Malice, is able to make a man bothe to believe lyes, and tell them; and sometimes also to invent them.”
― Thomas Hobbes, Leviathan

**Comment by AirFireWater** [comment]
“If you tell a lie big enough and keep repeating it, people will eventually come to believe it. The lie can be maintained only for such time as the State can shield the people from the political, economic and/or military consequences of the lie. It thus becomes vitally important for the State to use all of its powers to repress dissent, for the truth is the mortal enemy of the lie, and thus by extension, the truth is the greatest enemy of the State.”
 - Joseph Goebbels (apparently)

**Reply by dscnode** [reply]
https://truthisthegreatestenemyofthestate.blogspot.com/2012/04/outline-of-problem.html

**Comment by AirFireWater** [comment]
"...the earth, as being their mother, delivered them, and now, as if their land were their mother and their nurse, they ought to take thought for her and defend her against any attack and regard the other citizens as their brothers and children of the self-same earth...While all of you, in the city, are brothers, we will say in our tale, yet god, in fashioning those of you who are fitted to hold rule, mingled gold in their generation, for which reason they are the most precious—but in the helpers, silver, and iron and brass in the farmers and other craftsmen. And, as you are all akin, though, for the most part, you will breed after your kinds, it may sometimes happen that a golden father would beget a silver son, and that a golden offspring would come from a silver sire, and that the rest would, in like manner, be born of one another. So that the first and chief injunction that the god lays upon the rulers is that of nothing else are they to be such careful guardians, and so intently observant as of the intermixture of these metals in the souls of their offspring, and if sons are born to them with an infusion of brass or iron they shall by no means give way to pity in their treatment of them, but shall assign to each the status due to his nature and thrust them out among the artisans or the farmers. And again, if from these there is born a son with unexpected gold or silver in his composition they shall honor such and bid them go up higher, some to the office of guardian, some to the assistanceship, alleging that there is an oracle that the city shall then be overthrown when the man of iron or brass is its guardian." - "The Noble Lie" attributed to Socrates in Plato's myth or parable of the metals in Book III. (https://en.wikipedia.org/wiki/The_Republic_(Plato)#Book_III) ,

**Comment by AirFireWater** [comment]
“Masonry, like all the Religions, all the Mysteries, Hermeticism and Alchemy, conceals its secrets from all except the Adepts and Sages, or the Elect, and uses false explanations and misinterpretations of its symbols to mislead those who deserve only to be misled; to conceal the Truth, which it calls Light, from them, and to draw them away from it.” - Albert Pike - Morals and Dogma

**Comment by AirFireWater** [comment]
“Trust, in particular among allies, is a targeted vulnerability. As any international institution does, NATO relies on trust between its partners. Trust is based not only on respecting
some explicit and tangible agreements, but also on ‘invisible contracts,’ on sharing values,
which is not easy when such a proportion of allied nations have been fighting each other for
centuries. This has left wounds and scars creating a cognitive/information landscape that our
adversaries study with great care. Their objective is to identify the ‘Cognitive Centers of
Gravity’ of the Alliance, which they will target with ‘info-weapons’.”
- Cognitive Warfare

**Comment by AirFireWater** [comment]
"Hello ma'am, this is Walter from Microsoft. We had an issue at the office and we accidentally deposited $5000 into your account, we will need a moment of your time to sort this out" 
- Internet Scammer

**Comment by AirFireWater** [comment]
"Loose one inch of belly fat per week with **this one weird trick**" 
- clickbait advertisement

**Comment by PlainsDan** [comment]
“Beware of false prophets, who come to you in sheep’s clothing, but underneath are ravenous wolves."
Matthew 7:15

**Comment by siriusbboy** [comment]
Men must be taught as if you taught them not
And things unknown proposed as things forgot.
-Alexander Pope

**Comment by HyperCrowd** [comment]
Baltasar Gracian’s maxims was orientated towards "checkmating their will.” Love that phrase.

“_Treat your friends as though they could become your worst enemies. Since this happens in reality, let it happen in foresight._” - Baltasar Gracián

"_Skillfully make servants of those whom nature made superior._" - Baltasar Gracián

"_Do, but also seem._" - Baltasar Gracián

"_The person who tells his secrets to another makes himself a slave._" - Baltasar Gracián

“_Why expect others to guard our secrets if we cannot keep them ourselves._” - François de La Rochefoucauld

"_No mortal can keep a secret. If his lips are silent, he chatters with his fingertips; betrayal oozes out of him at every pore._" - Freud

“_Understanding power empowers._” - by Peter Sjöstedt-H

“_Mundus vult decipi, ergo decipiatur.” [The world wants to be deceived, so let it be deceived.]_"

"_If you don't have a strategy, you're part of someone else's strategy._" - Alvin Toffler

"_You may not be interested in war, but war is interested in you._" - Leon Trotsky

"_As soon as you have made a thought, laugh at it._" - Lao Tzu

"_The desire of the man is for the woman, but the desire of the woman is for the desire of the man._" - Madame de Stael

“_Masakatsu agatsu” [True victory is victory over the self.]_

"_My unconscious knows more about the consciousness of the psychologist than his consciousness knows about my unconscious._" - Karl Kraus

"_Your whole idea about yourself is borrowed--borrowed from those who have no idea of who they are themselves._" - Osho

"_One can deceive a person for the truth's sake, and (to recall old Socrates) one can deceive a person into the truth._” - Kierkegaard

"_Then we will no longer be immature like children. We won't be tossed and blown about by every wind of new teaching. We will not be influenced when people try to trick us with lies so clever they sound like the truth._" - Ephesians 4:14

“_The less we read, the more harmful it is what we read._” - Miguel de Unamuno

“_Man matures when he stops believing that politics solves his problems._” ― Nicolás Gómez Dávila

"_Hell can't be made attractive, so the devil makes attractive the road that leads there._" - St. Basil the Great

"_It is difficult to get a man to understand something when his salary depends upon his not understanding it._" - Upton Sinclair

“_God can save the sinner that I am, but not the saint I pretend to be._” - St John of Kronstadt

**Comment by freyjaCode** [comment]
[“The conscious and intelligent manipulation of the organized habits and opinions of the masses is an important element in democratic society. Those who manipulate this unseen mechanism of society constitute an invisible government which is the true ruling power of our country. ...We are governed, our minds are molded, our tastes formed, our ideas suggested, largely by men we have never heard of. This is a logical result of the way in which our democratic society is organized. Vast numbers of human beings must cooperate in this manner if they are to live together as a smoothly functioning society. ...In almost every act of our daily lives, whether in the sphere of politics or business, in our social conduct or our ethical thinking, we are dominated by the relatively small number of persons...who understand the mental processes and social patterns of the masses. It is they who pull the wires which control the public mind.”](https://www.goodreads.com/work/quotes/481391-propaganda)
― Edward Bernays, Propaganda

**Comment by freyjaCode** [comment]
[“One believes things because one has been conditioned to believe them.”](https://www.goodreads.com/work/quotes/3204877)
― Aldous Huxley, Brave New World

**Comment by HyperCrowd** [comment]
A breakdown of measurable psychosecurity:

https://docs.google.com/spreadsheets/d/1xeVbABlys47Q5GevC32X87QBshCvhkpFcfKvZU-caPA/edit?usp=sharing
